09 Redux/src/reducers/membersReducer.ts

```diff
@@ -0,0 +1,18 @@
+import MemberEntity from "../api/memberEntity";
+import MemberAPI from "../api/memberAPI";
+
+// Just to show how combine reducers work, we have
+// divided into two reducers member load + member load/update/delete
+export default (state : Array<MemberEntity> = [], action) => {
+  switch (action.type) {
+    case 'MEMBERS_LOAD':
+      let members : Array<MemberEntity>;
+      members = MemberAPI.getAllMembers();
+
+      return members;
```

Comment:
Mmm... would this operator just concat all the existing data into a new array?

http://ariya.ofilabs.com/2013/03/es6-and-spread-operator.html

I will apply it, but not sure if we should clone the whole object?